test/map-set-objects.spec.js

```diff
@@ -0,0 +1,133 @@
+import assert from "assert";
+
+describe("Map", function () {
+  it("Should explore CRUD methods in map", function () {
+    let sayings = new Map();
+    sayings.set("dog", "woof");
+    sayings.set("cat", "meow");
+    sayings.set("elephant", "toot");
+    sayings.set(1, "number");
+    console.log(sayings);
+    assert.strictEqual(sayings.size, 4);
+    assert.strictEqual(sayings.get("cat"), "meow");
+    assert.strictEqual(sayings.get("fox"), undefined);
+    assert.strictEqual(sayings.has("dog"), true);
+    assert.strictEqual(sayings.has("fox"), false);
+    sayings.delete("dog");
+    sayings.delete(1);
+    assert.strictEqual(sayings.has("dog"), false);
+    assert.strictEqual(sayings.has(1), false);
+    sayings.clear();
+    assert.strictEqual(sayings.size, 0);
+  });
+});
+
+describe("Set", function () {
```

Comment:
The primary use of `Set` in practical programming is to get unique values. I would recommend adding a test case along those lines. `new Set([1,1,1,2,2,2,3,3,3,4,4,4,4,4,4,5])`. You will find in many places in our code something like `const uniqueVals = Array.from(new Set(vals));`

Reply:
Added test